libs/database/src/lib/entities/site.entity.ts

```diff
@@ -96,6 +108,102 @@ export class Site extends Model<Site> {
   @Column(TEXT)
   descriptionSitingStrategy: string | null;
 
+  @AllowNull
+  @Column(INTEGER.UNSIGNED)
```

Comment:
According to the current DB schema, this should be a `DECIMAL(15, 1)`. Please carefully audit these fields against the current DB schema to make sure they all match. Once you've done that, I'll take another look over these new fields.